Lib/test/test_os.py

```diff
@@ -4085,9 +4086,16 @@ def test_eventfd_select(self):
 @unittest.skipUnless(hasattr(os, 'timerfd_create'), 'requires os.timerfd_create')
 @support.requires_linux_version(2, 6, 30)
 class TimerfdTests(unittest.TestCase):
-    # Tolerate a difference of 1 ms
-    CLOCK_RES_NS = 1_000_000
-    CLOCK_RES = CLOCK_RES_NS * 1e-9
+    if sys.platform == "android" and platform.android_ver().is_emulator:
+        # Tolerate a difference of 10 ms (1 ms is not reliably achievable on
```

Comment:
Can you mention Android emulator explicitly? Usually, we also mention the issue number: gh-108277.

Reply:
I've updated the comment.